### DISCUSSION TITLE
Geometry definition in LandCover data theme

### DISCUSSION THREAD

**Discussion by ulbrichtd** [INSPIRE-MIF/helpdesk#150] [Q&A] [answered]
Hi,

we have objects with Multipart geometry in our landcover data. Is is possible to use MultiSurface in attribute "geometry" of feature type LandCoverUnit? The IR stats that the attribute geometry is of type GM_Object and the INSPIRE validator currently does not accept MultiSurfaces.

The data theme ProtectedSites defines the attribute geometry also as GM_Object and the INSPIRE validator does accept Multipart objects.

What is the difference between these themes that requires a restriction in the geometry definition? 

Best regards and many thanks,
Birgit & Damian (from Berlin)

**Comment by fabiovinci** [comment, marked as answer]
Dear @ulbrichtd,

the LandCoverUnit feature type does not allow the use of Multipart geometries since there is a specific constraint in the Implementing Rules (IR 1253/2013).

![image](https://user-images.githubusercontent.com/22212406/231433097-448dd36d-3cac-4880-b342-3b552bf5be92.png)

The constraint is also documented in the UML diagram:

![image](https://user-images.githubusercontent.com/22212406/231425974-7608fb78-8706-4a98-a269-94bb171b276b.png)

Regarding the ProtectedSites feature type, the geometry data type is the same, i.e. GM_Object, but in this case, there are no constraints, so any geometry is allowed.

I hope that this helps to clarify.

**Comment by ulbrichtd** [comment]
Hi Fabio,
many thanks. This helps. 
Best regards
Damian
